client/app/components/dynamic-table/index.js

```diff
@@ -69,12 +69,12 @@ function createRowRenderTemplate(columns, $compile) {
     switch (column.displayAs) {
       case 'json':
         return `
-          <dynamic-table-json-cell column="columns[${index}]" 
+          <dynamic-table-json-cell column="columns[${index}]"
```

Comment:
Is it somehow related to this PR?

Reply:
An earlier linter was whining about this though it is otherwise unrelated. Will gladly remove the whitespace change if needed.

Reply:
Interesting. What errors did you get from linter?

Reply:
ping :slightly_smiling_face: I'm really wondering why there are any codestyle notices for this code - because that's string literal and any linter wont touch it.

Reply:
I am probably not remembering right. I think (in november of last year) there was something the linter required here (possibly unused import or something) and after I made the change my code editor removed the trailing whitespace. I will go into vi and re-add it.

Reply:
Which editor do you use? Because my one also removes trailing whitespaces, but keeps in multiline string literals (because in general case it's unsafe to remove them)

Reply:
And while it's not so important, but I think you should revert this - because the entire file is not related to changes declared in this PR.

Reply:
Yep agreed.